libnmstate/nm/bond.py

```diff
@@ -45,20 +55,38 @@ def is_bond_type_id(type_id):
 
 def get_bond_info(nm_device):
     slaves = get_slaves(nm_device)
-    options = get_options(nm_device)
+    options = _get_options(nm_device)
     if slaves or options:
         return {"slaves": slaves, "options": options}
     else:
         return {}
 
 
-def get_options(nm_device):
-    con = connection.ConnectionProfile()
-    con.import_by_device(nm_device)
-    if con.profile:
-        bond_settings = con.profile.get_setting_bond()
-        return bond_settings.props.options
-    return {}
+def _get_options(nm_device):
+    ifname = nm_device.get_iface()
+    if not ifname:
```

Comment:
Is that possible? If `nm_device.get_iface()` can be empty at this point maybe we have this issue in other interfaces but I don't think so..

Reply:
I tried 100 times with bond query after creation/deletion. Never get a empty iface name.
Hence these code removed.

Reply:
Thanks!